README.md

```diff
@@ -319,6 +319,116 @@ You can use dbt-coves to improve the standard dbt docs generation process. It ge
 # Mandatory when using --merge-deferred
 ```
 
+### Generate airflow-dags
+
+```console
+dbt-coves generate airflow-dags
+```
+
+Translate YML files into their Airflow Python code equivalent. With this, DAGs can be easily written with some `key:value` pairs.
+
+The basic structure of these YMLs must consist of:
+
+- Global configurations (description, schedule_interval, tags, catchup, etc.)
+- `default_args`
+- `nodes`: where tasks and task groups are defined
+  - each Node is a nested object, with it's `name` as key and it's configuration as values.
+    - this configuration must cover:
+      - `type`: 'task' or 'task_group'
+      - `operator`: Airflow operator that will run the tasks (full _module.class_ naming)
+      - `dependencies`: whether the task is dependent on another one(s)
+      - any `key:value` pair of [Operator arguments](https://airflow.apache.org/docs/apache-airflow/stable/_api/airflow/operators/index.html)
+
+#### Airflow DAG Generators
+
+When a YML Dag `node` is of type `task_group`, **Generators** can be used instead of `Operators`.
+
+They are custom classes that receive YML `key:value` pairs and return one or more tasks for the respective task group. Any pair specified other than `type: task_group` will be passed to the specified `generator`, and it has the responsibility of returning N amount of `task_name = Operator(params)`.
+
+We provide some prebuilt Generators:
+
+- `AirbyteGenerator` creates `AirbyteTriggerSyncOperator` tasks, which are used in Airflow to sync Airbyte connections
+
+  - It must receive Airbyte's `host` and `port` and a `connection_ids` list of Airbyte Connections to Sync
+
+- `FivetranGenerator`: creates `FivetranOperator` tasks, which are used to trigger Fivetran sync operations from Airflow
+  - It must receive Fivetran's `api_key`, `api_secret` and a `connection_ids` list of Fivetran Connectors to Sync. It can optionally receive `wait_for_completion: true` and 2 tasks will be created for each sync: a `FivetranOperator` and it's respective `FivetranSensor` that monitors the status of the sync.
+- `AirbyteDbtGenerator` and `FivetranDbtGenerator`: instead of passing them Airbyte or Fivetran connections, they use dbt to discover those IDs. Apart from their parent Generators mandatory fields, they can receive:
+  - `dbt_project_path`: dbt/project/folder
+  - `virtualenv_path`: path to a virtualenv in case dbt has to be ran with another Python executable
+  - `run_dbt_compile`: true/false
+  - `run_dbt_deps`: true/false
+
+#### Basic YML DAG example:
+
+```yaml
+description: "dbt-coves DAG"
+schedule_interval: "@hourly"
+tags:
+  - version_01
+default_args:
+  start_date: 2023-01-01
+catchup: false
+nodes:
+  sync_airbyte:
+    type: task_group
+    tooltip: "Sync Airbyte connections"
+    generator: AirbyteGenerator
```

Comment:
I'd use `AirbyteDbtGenerator` as the basic example. As Noel said, people won't know connection_ids so this example won't be used much